Error: Cannot find module 'msal'

# Bug Report

## Prerequisites

-   [x] Can you reproduce the problem?
-   [x] Are you running the latest version?
-   [x] Are you reporting to the correct repository?
-   [x] Did you perform a cursory search?

For more information, see the `CONTRIBUTING` guide.

## Description

Building my app in a docker container and running v2.2.0 of `@microsoft/microsoft-graph-client` in Node.js 14.13.1.

This is a new error that I didn't experience last week when building the container, but I didn't notice any recent changes to `package.json` or `ImplicitMSALAuthenticationProvider.ts` that would cause this.

**Console Errors:** [Is there any console error]

```
Error: Cannot find module 'msal'
Require stack:
- /usr/app/node_modules/@microsoft/microsoft-graph-client/lib/src/ImplicitMSALAuthenticationProvider.js
- /usr/app/node_modules/@microsoft/microsoft-graph-client/lib/src/index.js
```

**Screenshots:** [If applicable, add screenshots to help explain your problem]

N/A

## Steps to Reproduce

1. Add `@microsoft/microsoft-graph-client` to your Node.js project.
2. `import { Client } from '@microsoft/microsoft-graph-client'` from a file in your project
3. See error

**Expected behavior:** [What you expected to happen]

Not error

**Actual behavior:** [What actually happened]

Error above

## Additional Context

Add any other context about the problem here..

## Usage Information

SDK Version - [SDK version you are using]

v2.2.0

-   [x] Node (Check, if using Node version of SDK)

> Node Version - [The version of Node you are using]

14.13.1

-   [ ] Browser (Check, if using Browser version of SDK)

> Browser Name - [The name of Browser that you are using for SDK]

> Version - [The version of the browser you are using]

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
 	"name": "@microsoft/microsoft-graph-client",
-	"version": "2.2.0",
+	"version": "2.2.1",
 	"description": "Microsoft Graph Client Library",
 	"license": "MIT",
 	"main": "lib/src/index.js",

--- a/package.json
+++ b/package.json
@@ -13,6 +13,7 @@
 	"types": "./lib/src/index.d.ts",
 	"dependencies": {
 		"@babel/runtime": "^7.4.4",
+		"msal": "^1.4.4",
 		"tslib": "^1.9.3"
 	},
 	"devDependencies": {
@@ -27,7 +28,6 @@
 		"isomorphic-fetch": "^2.2.1",
 		"lint-staged": "^8.1.5",
 		"mocha": "^6.1.4",
-		"msal": "^1.0.0",
 		"prettier": "^1.17.0",
 		"rollup": "^1.10.1",
 		"rollup-plugin-babel": "^4.3.2",

--- a/src/Version.ts
+++ b/src/Version.ts
@@ -12,4 +12,4 @@
  * @module Version
  */
 
-export const PACKAGE_VERSION = "2.2.0";
+export const PACKAGE_VERSION = "2.2.1";

